sphinx/util/i18n.py

```diff
@@ -274,7 +274,7 @@ def format_date(format: str, date: datetime = None, language: str = None) -> str
         if source_date_epoch is not None:
             date = datetime.utcfromtimestamp(float(source_date_epoch))
         else:
-            date = datetime.utcnow()
+            date = datetime.now(timezone.utc).astimezone()
```

Comment:
Hmmm... I'm not sure this does what we expect it to do.

* If we want `date` to be set to a datetime in UTC, then I think we want this: `date = datetime.now().astimezone(timezone.utc)`.

* If we want `date` to be always be a non-naive datetime (i.e. uses the system timezone as the timezone), then I think we want this: `date = datetime.now().astimezone()`.

I think in either case, this will format as expected when passed to the `babel` library because the `date` datetime that babel now gets will not be naive: it will either be explicitly in UTC, or it will explicitly be in whatever the system timezone is.

Reply:
I didn't know about `datetime.now().astimezone()` ... that seems like a great improvement!